android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsState.kt

```diff
@@ -36,4 +36,5 @@ data class TranslationsState(
     val pageSettings: TranslationPageSettings? = null,
     val translationError: TranslationError? = null,
     val settingsError: TranslationError? = null,
+    val neverTranslateSites: List<String>? = null,
```

Comment:
nit: Please add the `@property` too, maybe also move above the property before the `Errors`?

Reply:
Added documentation and reordered the priorities